### PR TITLE
Use autoloading

### PR DESCRIPTION
**Pull request by annda**

### CODE REVIEW

File: syntax.php [hunk 1]
```diff
@@ -168,7 +164,6 @@ function handle($match, $state, $pos, Doku_Handler $handler){
             }
 
             // parse it
-            require_once(dirname(__FILE__).'/XMLParser.php');
```
Comment (splitbrain) [inline syntax.php:171]: Since this is part of the plugin and does not have a namespace, I don't think it will be autoloaded.

File: syntax.php [hunk 2]
```diff
@@ -8,10 +8,6 @@
 
 if(!defined('DOKU_INC')) define('DOKU_INC',realpath(dirname(__FILE__).'/../../').'/');
 if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
```
Comment (splitbrain) [inline syntax.php:10]: these two lines can be removed as well.

Comment (annda) [reply syntax.php]: done

File: syntax.php (hunk 1 above)
Comment (annda) [reply syntax.php]: fixed

**Pull request merged**